hugegraph-api/src/main/java/com/baidu/hugegraph/api/traversers/PathsAPI.java

```diff
@@ -114,9 +114,9 @@ public String post(@Context GraphManager manager,
                                "The targets of request can't be null");
         E.checkArgumentNotNull(request.step,
                                "The step of request can't be null");
-        E.checkArgument(request.depth > 0,
-                        "The depth of request must be > 0, but got: %s",
-                        request.depth);
+        E.checkArgument(request.depth > 0 && request.depth <= 5000,
+                        "The depth of request must be in (0, 5000], " +
+                        "but got: %s", request.depth);
```

Comment:
should also add depth check to other APIs?

Reply:
all path api use same limit?

Reply:
yes, it's better to keep consistent